src/js/common/utils/data-utils/set.js

```diff
@@ -11,9 +11,9 @@ import checkValidPath from './checkValidPath';
  *
  * TODO: Link to some documentation which will be a better explanation than the above
  *
- * @param {Array|Obect} object
  * @param {Array|string} path
  * @param {*} value
+ * @param {Array|Obect} object
```

Comment:
Typo

Reply:
Heh, these typos are just copy pasta from earlier (realized I changed the param order but never updated the documentation). :flushed:

Good eye! :male_detective: